peer/brontide.go

```diff
@@ -2178,7 +2220,13 @@ func (p *Brontide) writeMessage(msg lnwire.Message) error {
 	flushMsg := func() error {
 		// Ensure the write deadline is set before we attempt to send
 		// the message.
-		writeDeadline := time.Now().Add(writeMessageTimeout)
+		multiplier := 1
```

Comment:
Could roll this section and the above (read + write deadline) into a small helper function that does the multiplier mapping internally, and just returns the deadline to use.

Reply:
Looks like it can also be used above for the ping interval + timeouts.